modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -225,9 +225,9 @@ supports up to `64` threads. If this field is empty, the Ingress Controller uses
 
 * `tunnelTimeout` specifies how long a tunnel connection, including websockets, remains open while the tunnel is idle. The default timeout is `1h`.
 
-* `maxConnections` specifies the maximum number of simultaneous connections that can be established per HAProxy process. Increasing this value allows each ingress controller pod handle more connections at the cost of additional system resources. Permitted values are `0`, `-1`, any value within the range `2000` and `2000000`, or the field can be left empty. 
+* `maxConnections` specifies the maximum number of simultaneous connections that can be established per HAProxy process. Increasing this value allows each ingress controller pod to handle more connections at the cost of additional system resources. Permitted values are `0`, `-1`, any value within the range `2000` and `2000000`, or the field can be left empty. 
 
-** If this field is left empty or has the value `0`, the ingress controller will use the default value of `20000`. This value is subject to change in future releases.
+** If this field is left empty or has the value `0`, the ingress controller will use the default value of `50000`. This value is subject to change in future releases.
 
 ** If the field has the value of `-1`, then HAProxy will dynamically compute a maximum value based on the available `ulimits` in the running container. This process results in a large computed value that will incur significant memory usage compared to the current default value of `20000`.
```

Comment:
This line also mentions 20000 and should also be adjusted for 50000.

Reply:
Oof, sorry for missing this.